scripts/build-cl-polkadot.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR=`dirname "$BASH_SOURCE"`
+pushd $DIR
+
+# The following is fake and comes from CI
+source .env
```

Comment:
I wanted to test running this file on my machine but couldn't due to not having a `.env` and not knowing what to put in it. Is this something I should just not do, and leave it for CI?

Reply:
I added a `.env-sample`. This is bascially a CI cheap replacement...